volumectl

```diff
@@ -122,8 +122,10 @@ if [ "$dev_type" = 'source' ] && [ -z "$dev" ]; then
 elif [ "$dev" ]; then
 	dev_opt="--$dev_type=$dev"
 elif [ "$dev_type" = 'sink' ] && [ "$use_playing" ]; then
-	dev=$(pactl list short sinks | grep RUNNING | cut -f 2 | head -n 1)
-	dev_opt="--$dev_type=$dev"
+	dev="$(pactl list short sinks | { grep RUNNING || true; } | cut -f 2 | head -n 1)"
+	if [ "$dev" ]; then
+		dev_opt="--$dev_type=$dev"
+	fi
```

Comment:
This is ugly. The following should work, but didn't test it.

```
	if dev=$(pactl list short sinks | grep RUNNING | cut -f 2 | head -n 1) && [ "$dev" ]; then
 		dev_opt="--$dev_type=$dev"
 	fi
```

Reply:
Thanks for the suggestion. It seems to works, I just pushed a new commit with it.